Modules/_uuidmodule.c

```diff
@@ -13,10 +19,23 @@ py_uuid_generate_time_safe(void)
     res = uuid_generate_time_safe(out);
     return Py_BuildValue("y#i", (const char *) out, sizeof(out), res);
 }
+#else
+static PyObject *
+py_uuid_generate_time(void)
+{
+    uuid_t out;
+    uuid_generate_time(out);
+    return PyBytes_FromStringAndSize((const char *) out, sizeof(out));
+}
+#endif
 
 
 static PyMethodDef uuid_methods[] = {
+#ifdef HAVE_TIME_SAFE
     {"generate_time_safe", (PyCFunction) py_uuid_generate_time_safe, METH_NOARGS, NULL},
+#else
+    {"generate_time", (PyCFunction) py_uuid_generate_time, METH_NOARGS, NULL},
```

Comment:
There's no need to expose two different function names, IMO.

Reply:
> There's no need to expose two different function names, IMO.

It's not the same API:
* _uuid.generate_time_safe() returns (uuid, is_safe).
* _uuid.generate_time() only returns the uuid

Reply:
I'm proposing that it's the same API (see above) :-)